src/mina-signer/tests/client.test.ts

```diff
@@ -1,4 +1,4 @@
-import Client from '../dist/node/mina-signer/mina-signer.js';
+import Client from '../../../dist/node/mina-signer/mina-signer.js';
```

Comment:
did the dist folder move?

Reply:
No, I thought I had changed something with that but I just didn't realize you were intended to build mina-signer before running the tests. I should've read the docs.